doc/htmldoc/templates/index.html

```diff
@@ -51,22 +51,22 @@
       <div class="row">
         <div class="grid_3">
           <h1>Welcome to the NEST Simulator documentation!</h1>
-        <a href="installation/index.html" class="buttonlink">Install NEST</a>
-
+          <p>NEST is a simulator for spiking neural network models that focuses on the dynamics, size and structure of neural systems rather than on the exact morphology of individual neurons.</p>
```

Comment:
To be consistent with newer text we've been using. I'd suggest replacing the first sentence with the following:  
NEST is used in computational neuroscience to model and study behavior of large networks of neurons. The models describe single neuron and synapse behavior and their connections. Different mechanisms of plasticity can be used to investigate artificial learning and help to shed light on the fundamental principles of how the brain works.